praetorian_cli/ui/console/commands/marcus.py

```
@@ -58,7 +59,10 @@ def _cmd_marcus(self, args):
 
         self.console.print(f'[primary]Entering conversation mode[/primary] [dim](type "/back" to return)[/dim]')
         self.console.print(f'[dim]Commands: /back, /new, /query, /agent, or just chat[/dim]')
-        self.console.print(f'[dim]Context: {self.context.summary()}[/dim]\n')
+        self.console.print(f'[dim]Context: {self.context.summary()}[/dim]')
+        if not self.context.account:
+            self.console.print(f'[warning]No account set -- Marcus will query your personal account. Use "set account <email>" first.[/warning]')
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)